### DISCUSSION TITLE
changer les paramètres du fichier train.py

### DISCUSSION THREAD

**Discussion by SiwarRziguiING3** [ultralytics/yolov5#8262] [Q&A]
bonsoir, je veux trouvé les paramètres similaire a yolov4 dans yolov5 tel que  :
1.  max batches
2. filters

**Comment by glenn-jocher** [comment]
@SiwarRziguiING3 model architecture is here:
https://github.com/ultralytics/yolov5/blob/1156a323f8fc221d5c1a36c700e72e129419d187/models/yolov5s.yaml#L1-L48

Training parameters are in train.py argparser:
https://github.com/ultralytics/yolov5/blob/1156a323f8fc221d5c1a36c700e72e129419d187/train.py#L477-L520
